File: Fibonacci/fibonacci.rb
# Fibonnaci Sequence on Ruby

# git-repository: https://github.com/360macky/seqCodes
# author: Marcelo A.S.

# Solicitamos el término
puts "Fibonacci"
puts "Ingresa el número de términos de la secuencia de Fibonacci"
n = gets.chomp.to_i

# Restamos 4 numeros
n = n - 4

# Proceso
a = 1
b = 2

puts "++++++++++"

puts 0
puts a
puts a
puts b

i = 0

while i < n
    sum = a + b

    puts sum
    a = b
    b = sum
    i+=1

end

File: Fibonacci/fibonacci.py
"""
Fibonnaci Sequence on Python

git-repository: https://github.com/360macky/seqCodes.git/
author: Marcelo A.S.

"""
print("Fibonacci")

try:
    n = int(input("Ingresa el número de términos de la secuencia de Fibonacci: "))
except ValueError:
    print("Valor incorrecto")
    n = 0

n -= 4

a = 1
b = 2

print("0\n1\n1\n2")

i = 0

while i < n:
    sum_n = a + b
    print(sum_n)
    a = b
    b = sum_n
    i+=1